mousecursor.go

```diff
@@ -0,0 +1,55 @@
+package flutter
+
+import (
+	"errors"
+	"fmt"
+	"github.com/go-flutter-desktop/go-flutter/plugin"
+	"github.com/go-gl/glfw/v3.3/glfw"
+)
+
+const mousecursorChannelName = "flutter/mousecursor"
+
+// mousecursorPlugin implements flutter.Plugin and handles method calls to the
+// flutter/mousecursor channel.
+type mousecursorPlugin struct {
+	channel   *plugin.MethodChannel
+	messenger plugin.BinaryMessenger
+	window    *glfw.Window
+}
+
+var defaultMousecursorPlugin = &mousecursorPlugin{}
+
+var _ PluginGLFW = &mousecursorPlugin{} // compile-time type check
+
+func (p *mousecursorPlugin) InitPlugin(messenger plugin.BinaryMessenger) error {
+	p.messenger = messenger
+
+	return nil
+}
+func (p *mousecursorPlugin) InitPluginGLFW(window *glfw.Window) error {
+	p.window = window
+	p.channel = plugin.NewMethodChannel(p.messenger, mousecursorChannelName, plugin.StandardMethodCodec{})
+	p.channel.HandleFuncSync("activateSystemCursor", p.handleActivateSystemCursor)
+	return nil
+}
+
+func (p *mousecursorPlugin) handleActivateSystemCursor(arguments interface{}) (reply interface{}, err error) {
```

Comment:
In the glfw doc there is https://godoc.org/github.com/go-gl/glfw/v3.3/glfw#Cursor.Destroy
Should we use it?

Reply:
Yes